app/nagbot.py

```diff
@@ -44,41 +39,40 @@ def notify_internal(channel, dryrun):
             num_active_resources = sum(1 for r in resources if r.is_active())
             num_total_resources = len(resources)
 
-            running_monthly_cost = money_to_string(sum(r.monthly_price for r in resources
-                                                       if r.included_in_monthly_price()))
+            running_monthly_cost = util.money_to_string(sum(r.monthly_price for r in resources
+                                                            if r.included_in_monthly_price()))
             summary_msg += f"\n*{resource_type.__name__}s:*\nWe have {num_active_resources} " \
                            f"{ec2_state} {ec2_type}s right now and {num_total_resources} total.\n" \
                            f"If we continue to run these {ec2_type}s all month, it would cost {running_monthly_cost}.\n"
 
             resources = sorted((r for r in resources if not (len(r.eks_nodegroup_name) > 0)), key=lambda i: i.name)
 
-            resources_to_terminate = (list(r for r in resources if r.is_terminatable(TODAY_YYYY_MM_DD)))
-            resources_to_stop = list(r for r in resources if r.is_stoppable(today_date=TODAY_YYYY_MM_DD))
+            resources_to_terminate = (list(r for r in resources if r.can_be_terminated()))
+            resources_to_stop = list(r for r in resources if (r.can_be_stopped()))
 
             if len(resources_to_terminate) > 0:
                 summary_msg += f'The following {len(resources_to_terminate)} {ec2_type}s are due to be *TERMINATED*, ' \
                                'based on the "Terminate after" tag:\n'
                 for r in resources_to_terminate:
                     contact = sqslack.lookup_user_by_email(r.contact)
-                    summary_msg += r.make_resource_summary() + \
-                        f', "Terminate after"={r.terminate_after}, "Monthly Price"={money_to_string(r.monthly_price)}' \
-                        f', Contact={contact}\n'
-                    resource.set_tag(r.region_name, r.ec2_type, r.resource_id, r.terminate_after_tag_name,
-                                     parsing.add_warning_to_tag(r.terminate_after, TODAY_YYYY_MM_DD), dryrun=dryrun)
+                    summary_msg += r.make_resource_summary() + f', "Terminate after"={r.terminate_after}, ' \
+                        f'"Monthly Price"={util.money_to_string(r.monthly_price)}, Contact={contact}\n'
+                    util.set_tag(r.region_name, r.ec2_type, r.resource_id, r.terminate_after_tag_name,
+                                 parsing.add_warning_to_tag(r.terminate_after, util.TODAY_YYYY_MM_DD), dryrun=dryrun)
             else:
                 summary_msg += f'No {ec2_type}s are due to be terminated at this time.\n'
-
-            if resource_type.can_be_stopped():
+            # only 'instance' can be stopped
+            if ec2_type.lower() == 'instance':
```

Comment:
I think some changes might have been lost in merge? I think having the type define if it can be stopped is a better pattern